plonetheme/onegovbear/theme/scss/globals/variables.scss

```diff
@@ -0,0 +1,16 @@
+/*
+  Fonts
+ */
+$font-family-primary: "HelveticaNeue-Light", "Helvetica Neue Light", "Helvetica Neue", Helvetica, Arial, "Lucida Grande", sans-serif;
+$font-family-heading: $font-family-primary;
+$font-family-code: "Courier New", Courier, monospace;
+
+$color-page-background: $color-gray-light !default;
+$color-content-background: $color-white !default;
+
+@include declare-variables(
+  $font-family-primary,
+  $font-family-heading,
```

Comment:
Please remove the leading `$`s, otherwise the value of the variable is registered rather than the name of the variable:
<img width="1132" alt="bildschirmfoto 2016-01-29 um 11 17 48" src="https://cloud.githubusercontent.com/assets/7469/12672946/f5a4f1ce-c679-11e5-8e11-210764082cad.png">